packages/client/auth-core/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export { useAuth, AuthProvider, type AuthProviderParams } from "./contexts/authlogin";
```

Comment:
We've removed the distinction between server/client here and just have a single file for exporting everything.

Reply:
This fixed some build issues and simplifies things for now, can be reversed later if the need arises